### ISSUE TITLE
HTTPS not supported

### ISSUE THREAD

**Issue by mko** [watilde/tvm#37]
Performing standard install of latest version of TypeScript using `tvm` via the `tvm install latest` results in an HTTPS error.

```
_http_client.js:55
    throw new Error('Protocol "' + protocol + '" not supported. ' +
    ^

Error: Protocol "https:" not supported. Expected "http:"
    at new ClientRequest (_http_client.js:55:11)
    at Object.exports.request (http.js:31:10)
    at Object.exports.get (http.js:35:21)
    at /usr/local/lib/node_modules/tvm/lib/abstract/install_latest.js:37:10
    at fn (/usr/local/lib/node_modules/tvm/node_modules/async/lib/async.js:746:34)
    at /usr/local/lib/node_modules/tvm/node_modules/async/lib/async.js:1213:16
    at /usr/local/lib/node_modules/tvm/node_modules/async/lib/async.js:166:37
    at /usr/local/lib/node_modules/tvm/node_modules/async/lib/async.js:706:43
    at /usr/local/lib/node_modules/tvm/node_modules/async/lib/async.js:167:37
    at Immediate.<anonymous> (/usr/local/lib/node_modules/tvm/node_modules/async/lib/async.js:1206:34)
```

**Comment by watilde** [comment]
Wow thanks for reporting this! It's a bug. Will fix it :)

**Comment by watilde** [comment]
Released as v0.8.12. Thanks!
